### ISSUE TITLE
Cant use the same glyph with Dual Spec

### ISSUE THREAD

**Issue by Slumwawa** [SunwellTracker/issues#525]
Decription:  I can't use minor Glyph of Unburdened Rebirth in both on my dual specs.  When i try it says "that glyph is already inscribed in your spellbook". 

How it works:  I have one minor Glyph of Unburdened Rebirth for my tanking spec but it wont let me put one minor Glyph of Unburdened Rebirth into my DPS spec when i switch to that spec. 

How it should work: 
I should be able to use minor Glyph of Unburdened Rebirth in both my specs. 

Source (you should point out proofs of your report, please give us some source):  i should be able to use the same glyph in both specs.

**Comment by Cwiercpasiec** [comment]
If you put a glyph in one spec, then change your active spec and try tu put the same glyph in your other spec, you will get an error saying "That Glyph is already inscribed in your spellbook." and won't be able to inscribe glyph until you relog. After relog everything works properly, you can use the glyph that is already inscribed in your inactive build. I also tried respecing to the build with glyphs, then again to the one without glyphs and I could put the same glyphs as in inactive spec.
In conclusion, to use the same glyph in both specs you simply have to inscribe one glyph, log out, log in and at this time everything should be ok. Minor bug, but quite irritating one.

**Comment by sitowsky** [comment]
There is no source in this report. Please find a source(proof) and report bug properly once again. 

Thanks in advance.
